include/csv.h

```diff
@@ -0,0 +1,1280 @@
+// Copyright: (2012-2015) Ben Strasser <code@ben-strasser.net>
+// License: BSD-3
```

Comment:
in https://github.com/ceph/ceph/blob/master/COPYING, the Ceph project has been trying to document the copyright and license information for all of the source that we include this way. we should update that COPYING file to credit this as well, since it'll be included in Ceph via submodule

Reply:
p.s. @galsalomon66 i see that s3select doesn't specify its license, and it should! please pick something compatible with ceph's LGPL license, and put it in a `LICENSE` or `COPYING` file. github can even automate that https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository

Reply:
> p.s. @galsalomon66 i see that s3select doesn't specify its license, and it should! please pick something compatible with ceph's LGPL license, and put it in a `LICENSE` or `COPYING` file. github can even automate that https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository

i did add LICENSE more than a year ago (https://github.com/galsalomon66/s3select/blob/master/LICENSE)
CEPH/s3select fork from galsalomon66/s3select, it's not enough?

Reply:
ok, thanks. i guess we just need to rebase ceph/s3select on top of galsalomon66/s3select? does your repo have any other important changes that ceph/s3select doesn't?